templates/compute.yaml.j2

```diff
@@ -274,7 +274,7 @@ resources:
                               -
                                 ip_netmask:
                                     get_param: StorageIpSubnet
```

Comment:
Add TenantIpSubnet here in case of vlan network

Reply:
if we set vlan_provider_network = true, it will be added here, since the below condition fails